### PR TITLE
Added implementations of CRUD methods and their tests for FacultyRepo…

### PR DESCRIPTION
**Pull request by SergeyPrudnikov-dev**
…sitory, StatementByEntrantRepository, EnrolledPersonRepository.

### CODE REVIEW

File: src/test/java/com/epam/university/commission/repository/impl/EnrolledPersonRepositoryIntegrationTest.java [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+
+    private static final String USER_LOGIN = "test_login";
+    private static final String USER_EMAIL = "test_email@gmail.com";
+    private static final String USER_PASSWORD = "test_password";
+    private static UserRepository sUserRepository = new UserRepository();
+    private User user;
+
+    @BeforeEach
+    void setup() {
+
```
Comment (andrei-kuzniatsou) [inline src/test/java/com/epam/university/commission/repository/impl/EnrolledPersonRepositoryIntegrationTest.java:65]: remove empty lines

File: src/test/java/com/epam/university/commission/repository/impl/StatementByEntrantRepositoryIntegrationTest.java [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+/**
+ * Test to verify the functionality of the {@link StatementByEntrantRepository}.
+ * <p/>
+ * University-commission 2021  epam.com
+ * <p/>
+ * Date: 05/20/2021
+ *
+ * @author Siarhei Prudnikau1
+ */
+
```
Comment (andrei-kuzniatsou) [inline src/test/java/com/epam/university/commission/repository/impl/StatementByEntrantRepositoryIntegrationTest.java:26]: remove empty line

File: src/test/java/com/epam/university/commission/repository/impl/EnrolledPersonRepositoryIntegrationTest.java (hunk 1 above)
Comment (SergeyPrudnikov-dev) [reply src/test/java/com/epam/university/commission/repository/impl/EnrolledPersonRepositoryIntegrationTest.java:65]: fixed

File: src/test/java/com/epam/university/commission/repository/impl/StatementByEntrantRepositoryIntegrationTest.java (hunk 2 above)
Comment (SergeyPrudnikov-dev) [reply src/test/java/com/epam/university/commission/repository/impl/StatementByEntrantRepositoryIntegrationTest.java:26]: fixed

**Pull request merged**